packages/repo-tools/src/commands/index.ts

```diff
@@ -18,31 +18,83 @@ import { assertError } from '@backstage/errors';
 import { Command } from 'commander';
 import { exitWithError } from '../lib/errors';
 
-function registerSchemaCommand(program: Command) {
+function registerPackageCommand(program: Command) {
   const command = program
+    .command('package [command]')
+    .description('Various tools for working with specific packages.');
+
+  const schemaCommand = command
     .command('schema [command]')
-    .description('Various tools for working with API schema');
+    .description(
+      "Various tools for working with specific packages' API schema",
+    );
 
-  const openApiCommand = command
+  const openApiCommand = schemaCommand
     .command('openapi [command]')
-    .description('Tooling for OpenApi schema');
+    .description('Tooling for OpenAPI schema');
 
   openApiCommand
-    .command('verify [paths...]')
+    .command('init')
```

Comment:
Should have `<paths...>` as well right?

And this isn't really a package command right? Looks like it's meant to be run from the repo root

Reply:
My goal is for you to be able to run `init` in a single package to start onboarding to the tooling. I wouldn't _expect_ people to onboard multiple packages at the same time. diff here is a little weird, `verify` should be still repo level.

Reply:
Updated in https://github.com/backstage/backstage/pull/22350/commits/2a42f97175f2c04e7a4a60ea24150a8156a24ac3